Dockerfile

```diff
@@ -45,7 +45,7 @@ ARG TNF_SRC_URL=https://github.com/test-network-function/test-network-function
 ARG GIT_CHECKOUT_TARGET=$TNF_VERSION
 
 # Git identifier to checkout for partner
-ARG TNF_PARTNER_VERSION
+ARG TNF_PARTNER_VERSION=main
```

Comment:
this is not needed?

Reply:
before this change, we did not specify the branch, I guess it's defined as parameter when building the image, we can revert this change as well

Reply:
what do you think ?

Reply:
anyway, I'll wait for CI to pass. then remove it again and see the results.